package.json

```
@@ -1,5 +1,5 @@
 {
-    "name": "eslint-config-scality",
+    "name": "@scality/guidelines",
```

I'd rather call @scality/eslint-config or backend as there is as well a frontend config somewhere


We should also have 2 packages, 1 for js and 1 for ts

---

Sounds good for the package name:

```suggestion
    "name": "@scality/eslint-config", // or scality/eslint-backend-config
```

Any clue to give into where to find this frontend config package we have? I don't recall it when I was building the list of repo to upgrade.

As to the two packages story, in my understanding, when we have typescript code we usually "build" them (something I don't believe I'm doing right now) and include both the js and the Typescript declaration files in a single package. Is my assumption correct, am I missing something or did I misunderstood what you meant here?

---

No actually I don't see any package for the frontends

___

For now typescript packages are built via the postinstall script when you install with yarn.

But the thing is there are different rules with different parser for typescript project so we are supposed to turn off and replace js linting rules with typescript linting (like done in scuba). So typescript projects should not use this package but we should make another one using typescript rules.